arm-network/2017-06-01/swagger/loadBalancer.json

```diff
@@ -557,10 +558,11 @@
         },
         "protocol": {
           "type": "string",
-          "description": "The transport protocol for the endpoint. Possible values are: 'Udp' or 'Tcp'",
+          "description": "The transport protocol for the endpoint. Possible values are: 'Udp,' 'Tcp,' or 'All.'",
           "enum": [
             "Udp",
-            "Tcp"
+            "Tcp",
+            "All"
```

Comment:
Wouldn't it only be a breaking change if we removed one of the preexisting values?